app/src/main/assets/chart.html

```diff
@@ -7,6 +7,18 @@
 <script src="jquery-1.5.1.min.js"></script>
 <script src="freezeheader.js"></script>
 <script src="chart.js"></script>
+<script>
+  function isRowEmpty(parentNode) {
```

Comment:
Actually, can we do this in pebble instead of in JS? It would fit better with our current programming model that way, and I think it would be more performant.